### ISSUE TITLE
Missing data when import memory/table

### ISSUE THREAD

**Issue by AY268** [bytecodealliance/wasm-micro-runtime#3990]
I tried to import memory/table into a Wasm instance using the function wasm_instance_new_with_args, but I found that WAMR does not copy the data in the memory/table. I wonder how I could retain the data in the memory/table.

**Comment by lum1n0us** [comment]
Yes, the main branch of WAMR currently does not support importing and exporting memory and tables.

The development of fully spec-aligned import/export functionality is ongoing in a separate branch, dev/instantiate_linking.
